packages/fxa-content-server/app/tests/spec/lib/router.js

```diff
@@ -553,7 +608,7 @@ describe('lib/router', () => {
       });
       it('renders Backbone view when React conditions are not met', () => {
         routeHandler = router.createReactOrBackboneViewHandler(
-          'cannot_create_account',
+          'reset_password',
```

Comment:
Had to update this since `cannot_create_account` is in `simpleRoutes` which always shows the React version now.